src/authentication/basic_auth.py

```diff
@@ -54,6 +56,13 @@ def register_user(self, username, email, password, invite, **kwargs):
         token = user.email_token
         if not settings.MAIL["SEND"]:
             user.email_verified = True
+
+        if not config.get("enable_teams"):
+            team = Team(name=username, password=secrets.token_hex(32), owner=user)
```

Comment:
I feel like we should make secrets.token_hex(32) the default value for the password field, but thats a discussion for another day